src/availability.rs

```diff
@@ -623,7 +661,25 @@ mod test {
                 block,
                 client.get(&format!("block/{}", i)).send().await.unwrap()
             );
+            let block_summary: BlockSummaryQueryData<MockTypes> = client
```

Comment:
For consistency I think we should move the new queries and assertions into `validate` with the rest of the assertions